build/hyperkube.sh

```diff
@@ -43,3 +43,7 @@ docker run \
 
 docker run -d --net=host --privileged gcr.io/google_containers/hyperkube:v${K8S_VERSION} \
     /hyperkube proxy --master=http://127.0.0.1:${PORT} --v=2
+
+# Runs Heapster in standalone mode which listen on port 8082
+docker run --net=host -d kubernetes/heapster \
```

Comment:
Btw, can you specify the port here manually? This is not to rely on default port which may change